compiler/rustc_target/src/abi/call/mod.rs

```diff
@@ -652,6 +652,11 @@ impl<'a, Ty> ArgAbi<'a, Ty> {
         }
     }
 
+    /// Same as make_indirect, but doesn't check the current `PassMode`.
```

Comment:
Would be good to give some kind of guidance when it is okay to use this. Presumably `make_indirect` has these checks for a reason?